### PR TITLE
Make databricks an Enterprise feature

### PR DESCRIPTION
**Pull request by FlorianJacta**
Part of https://github.com/Avaiga/taipy-enterprise/issues/404

### CODE REVIEW

File: docs/tutorials/integration/4_databricks/index.md [hunk 1]
```diff
@@ -12,222 +12,14 @@ into interactive, production-ready applications for end-users. This article pres
 best practice for integrating both platforms: calling Databricks jobs from a Taipy 
 application.
```
Comment (jrobinAV) [inline docs/tutorials/integration/4_databricks/index.md]:
```suggestion
```
This article does not present anything now.

File: docs/tutorials/integration/4_databricks/index.md [hunk 2]
```diff
@@ -12,222 +12,14 @@ into interactive, production-ready applications for end-users. This article pres
 best practice for integrating both platforms: calling Databricks jobs from a Taipy 
 application. 
 
-More specifically, this article demonstrates how to integrate Taipy scenarios with 
-Databricks jobs. Taipy’s scenarios serve as a potent tool for orchestrating tasks and 
-performing ‘what-if ’analysis (i.e. examining various versions of a business problem). 
+!!! warning "Available in Taipy Enterprise edition"
 
+    The documentation and classes for connecting Databricks and Taipy are provided as
```
Comment (jrobinAV) [inline docs/tutorials/integration/4_databricks/index.md]:
```suggestion
    The documentation pages for connecting Taipy and Databricks are provided as 
```

File: docs/tutorials/integration/4_databricks/index.md (hunk 1 above)
Comment (FlorianJacta) [reply docs/tutorials/integration/4_databricks/index.md]: Well, for me, the article exists but is not available to the public. But that works for me

Comment (jrobinAV) [reply docs/tutorials/integration/4_databricks/index.md]: You already say it below, actually.

**Review by jrobinAV — APPROVED** [review]

**Review by jrobinAV — APPROVED** [review]

**Review by jrobinAV — COMMENTED** [review]
Can you check the quote line 209?

**Pull request merged**